app/templates/eg019_access_code_authentication.html

```diff
@@ -0,0 +1,43 @@
+<!-- extend base layout --> {% extends "base.html" %}  {% block content %}
+
+<h4>19. Send an envelope using an Access-code for multi-factor recipient authentication</h4>
+<p>The envelope includes a pdf, Word, and HTML document. Anchor text 
+  (<a href="https://support.docusign.com/en/guides/AutoPlace-New-DocuSign-Experience">AutoPlace</a>)
+  is used to position the signing fields in the documents.
+</p>
+<p>This is a general example of creating and sending an envelope (a signing request) to a recipient that requires an Access Code to open</p>
+
+  {% if show_doc %}
+    <p><a target='_blank' href='{{ documentation | safe }}'>Documentation</a> about this example.</p>
+  {% endif %}
+  
+<p>
+  The access-code to utilize the example provided is: <strong>NJ9@D1</strong>
```

Comment:
Would recommend if this is hard-coded to put it in a variable. This has to match what is in the .py file, right?